src/ngx_http_lua_shdict.c

```diff
@@ -2983,6 +2983,41 @@ ngx_http_lua_ffi_shdict_set_expire(ngx_shm_zone_t *zone, u_char *key,
 
     return NGX_OK;
 }
+
+#if nginx_version >= 1011007
+void
+ngx_http_lua_ffi_shdict_get_stats(ngx_shm_zone_t *zone,
+    size_t *total_used, size_t *total_size)
+{
+    ngx_http_lua_shdict_ctx_t   *ctx;
+    ngx_slab_stat_t             *stat;
+    size_t                       slot_count;
+    size_t                       slot_size;
+    size_t                       i;
+    size_t                       sum_used = 0;
+    size_t                       sum_size = 0;
+
+    ctx = zone->data;
+
+    ngx_shmtx_lock(&ctx->shpool->mutex);
```

Comment:
Better add a blank line after `ngx_shmtx_lock()` to make this more outstanding.

Reply:
done